exchange/settings/default.py

```diff
@@ -793,5 +779,26 @@ def filter_django_warnings(record):
 GOOGLE_ANALYTICS_ID = os.getenv('GOOGLE_ANALYTICS_ID', None)
 
 MAPLOOM_ENABLED = str2bool(os.getenv('MAPLOOM_ENABLED', 'True'))
+
+if MAPLOOM_ENABLED:
+    INSTALLED_APPS = INSTALLED_APPS + (
+        'exchange.maploom',
+    )
+    # static files storage
+    STATICFILES_DIRS += [
```

Comment:
```suggestion
    STATICFILES_DIRS = [
```
If you're just adding maploom/static to the list you don't need to do both `+=` and `+ STATICFILES_DIR`

Reply:
done